projects/optic/src/commands/capture/actions/captureRequests.ts

```diff
@@ -195,14 +195,18 @@ async function runRequestsCommand(
     reqCmd.on('exit', (code) => {
       if (code === 0) {
         resolve();
+      } else if (code) {
+        process.exitCode = code;
+        logger.error(`command ${command} failed with exit code ${code}`);
+        resolve();
       } else {
         reject();
       }
     });
   });
 
   reqCmd.stdout.on('data', (data) => {
-    logger.debug(data.toString());
+    logger.info(data.toString());
```

Comment:
I think it's correct here to long data as it comes out in `info` - where server only logs on `debug`. Reasoning here is that the command _may_ also be tests that you want to see the test output in. 

We might also need to do some formatting / handle interactions with the spinner here, but I think that can come later